uat/pom.xml

```diff
@@ -62,5 +62,8 @@
         <spotbugs.plugin.version>4.0.0</spotbugs.plugin.version>
         <antrun.plugin.version>3.0.0</antrun.plugin.version>
         <surefire.plugin.version>3.0.0</surefire.plugin.version>
+        <junit.version>5.9.2</junit.version>
```

Comment:
In the latest uat/pom.xml  junit.version and mockito.version are already defined. Did you rebase your branch on top of latest uat-dev ?

Reply:
Strictly speaking junit.version already defined in line 48
nd mockito.version on line 49

Reply:
Will be fixed in the next version )